ShareX/Forms/QRCodeForm.cs

```diff
@@ -137,6 +137,11 @@ private void DecodeImage(Bitmap bmp)
             if (results != null)
             {
                 output = string.Join(Environment.NewLine + Environment.NewLine, results);
+
+                if (UploadersLib.ResponseHelpers.UrlRedirectLink(output) != null)
+                {
+                    output = output + Environment.NewLine + Environment.NewLine + "This link redirects to: " + UploadersLib.ResponseHelpers.UrlRedirectLink(output);
```

Comment:
If you could please add this string to translation files (just do the ones you know - no google translate) (the files are at https://github.com/ShareX/ShareX/blob/develop/ShareX/Properties, the Resource.LANG.resx files) it would be appreciated

Reply:
I had deleted the repo from my computer and I'm having problems with committing/merging now that I've re-installed - is it easier to just cancel this whole PR and open a new one with revisions included? I've updated the Resources but have failing commits now